src/User.js

```diff
@@ -1,6 +1,21 @@
 class User {
-  constructor() {
-    
+  constructor(userDetails) {
```

Comment:
maybe if we pass in something that isn't an object it shouldn't set any properties